src/currency-field/CurrencyField.ts

```diff
@@ -193,17 +132,38 @@ export class CurrencyField extends OmniFormElement {
         return cleanValue;
     }
 
-    _formatToCurrency(preFormattedValue: number): string {
+    // Format the numeric value to a currency formatted string value.
+    _formatToCurrency(preFormattedValue: number | string): string {
+        if (preFormattedValue === 0) {
+            return preFormattedValue.toString();
+        }
+
         if (!preFormattedValue) {
+            console.log('No value provided');
```

Comment:
Do we want console logging in our components like this?

Reply:
Removing as requested.